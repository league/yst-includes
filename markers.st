
We can select by markers in the code.

~~~~ {include="selectlines.c" begin="// BEGIN" end="// END"}
~~~~

Testing begin only

~~~~ {include="marker1.c" begin="--{CUT}--"}
~~~~

