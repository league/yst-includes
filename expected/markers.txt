We can select by markers in the code.

~~~~
  int y = x + 2;
  printf("%d %d\n", x, y);
  printf("OK\n");
  return 3.14;
~~~~

Testing begin only

~~~~
int main() {
  printf("Foo!\n");
  return 0;
}
~~~~

Testing end only.

~~~~
#include <time.h>
#include <stdio.h>
#include <stdlib.h>

int main() {
  printf("Foo!\n");
  return 0;
}
~~~~
